changelogs/fragments/322-docs-README-updates.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Since, documentation changes are only made, the changelog is not needed and can be removed, or the changelog section could be changed as below:
```suggestion
trivial:
```

Reply:
Done.